src/pathrewriter.ts

```diff
@@ -24,15 +24,18 @@ import constants from './constants';
 
 const pathPosix = path.posix;
 
-class PathResolver {
-  constructor(public abspath?: boolean) {
+class PathRewriter {
```

Comment:
I left this comment on another commit I guess, but I hate how github shows these things: Can we get comments on this class and it's methods?